privacy_policy.html

```diff
@@ -0,0 +1,236 @@
+<p>La sécurité de vos données nous tient à cœur. Toutefois, n'oubliez pas qu'aucune méthode de transmission de données
+    par Internet ou méthode de stockage électronique n'est sûre à 100 %. Bien que nous nous efforçons d'utiliser des
+    méthodes commercialement acceptables pour protéger vos Données à Caractère Personnel, nous ne pouvons pas leur
+    garantir une sécurité absolue.</p>
+
+<h2>Prestataires de services</h2>
+<p>Nous pouvons faire appel à des sociétés tierces et à des tierces personnes pour faciliter la prestation de notre
+    Service ("Prestataires de Services"), assurer le Service en notre nom, assurer des services liés au Service ou nous
+    aider à analyser la façon dont notre Service est utilisé.</p>
+<p>Ces tiers n'ont accès à vos Données à Caractère Personnel que pour effectuer ces tâches en notre nom et il leur est
+    interdit de les communiquer ou de les utiliser à quelle qu'autre fin. </p>
+
+<h3>Analyses</h3>
+<p>Nous pouvons faire appel à des Prestataires de Services tiers pour surveiller et analyser l'utilisation de notre
+    Service.</p>
+<ul>
+    <li>
+        <p><strong>Google Analytics</strong></p>
+        <p>Google Analytics est un service d'analyse web proposé par Google qui assure le suivi du trafic d'un site web
+            et en rend compte. Google utilise les données recueillies pour suivre et surveiller l'utilisation de notre
+            Service. Ces données sont partagées avec d'autres services Google. Google peut utiliser les données
+            recueillies pour contextualiser et personnaliser les annonces de son propre réseau publicitaire.</p>
+        <p>Pour plus de précisions sur les pratiques de confidentialité de Google, merci de consulter la page web
+            Protection de la vie privée et conditions de Google: <a href="https://policies.google.com/privacy?hl=en">https://policies.google.com/privacy?hl=en</a>
+        </p>
+    </li>
+    <li>
+        <p><strong>Firebase</strong></p>
+        <p>Firebase est un service d'analyse fourni par Google Inc.</p>
+        <p>Vous pouvez refuser certaines fonctionnalités de Firebase en utilisant les paramètres de votre dispositif
+            mobile, par exemple ses paramètres publicitaires, ou en suivant les instructions qui figurent dans la
+            politique de confidentialité de Google: <a href="https://policies.google.com/privacy?hl=en">https://policies.google.com/privacy?hl=en</a>
+        </p>
+        <p>Nous vous encourageons également à consulter la politique de Google relative à la protection de vos données:
+            <a href="https://support.google.com/analytics/answer/6004245">https://support.google.com/analytics/answer/6004245</a>.
+        </p>
+        <p>Pour plus de précisions sur le type de données recueillies par Firebase, merci de consulter la page web
+            Protection de la vie privée et conditions de Google <a href="https://policies.google.com/privacy?hl=en">https://policies.google.com/privacy?hl=en</a>
```

Comment:
Shouldn't we link to a more firebase specific page :
https://support.google.com/firebase/answer/6318039
or
https://firebase.google.com/support/privacy/ ?

Reply:
maybe